src/core/Expression.cc

```diff
@@ -234,53 +234,33 @@ Range::Range(Expression *begin, Expression *step, Expression *end, const Locatio
 {
 }
 
-/**
- * This is separated because both PRINT_DEPRECATION and PRINT use
- * quite a lot of stack space and the method using it evaluate()
- * is called often when recursive functions are evaluated.
- * noinline is required, as we here specifically optimize for stack usage
- * during normal operating, not runtime during error handling.
- */
```

Comment:
We should leave that comment for the other print method.

Reply:
Someone who is more C++ savvy than I am should consider whether that comment is still accurate, given that Range::evaluate() and LOG() are their own functions.  In C, there would be no stack concern.

Reply:
I don't quite understand that comment, and also not sure how those functions use "quite a lot of stack space", neither now or when the comment was written. Not sure it's worth taking any action on it now anyway.